ui/src/experiments/list/ListExperimentsTable.js

```diff
@@ -88,11 +91,12 @@ const ListExperimentsTable = ({
       width: "5%",
       render: (item) => {
         return (
+          // We need to do urlJoin to correctly handle trailing slash when used as a remote component
           <EuiLink
             onClick={(e) => {
               e.stopPropagation();
             }}
-            href={item.id}
+            href={urlJoin(location.pathname, item.id)}
```

Comment:
Likewise for this - https://github.com/caraml-dev/xp/pull/45/files#diff-bad9138e8905bf7269b6399a5d022254ef273007f923a95f8e08ca199148ac2aL96.

Reply:
The previous change worked from the xp app, not when accessed as a remote component. Something that I hadn't checked when testing locally.